src/infrastructure/arranging/ordering.rs

```diff
@@ -128,12 +129,14 @@ impl<'a, T> ArrangingSystem for OrderingSystem<T> {
 /// - See [super-module](../index.html) for more information about streams.
 pub struct OrderingStream<T> {
     // the id of this stream.
-    stream_id: u8,
+    _stream_id: u8,
```

Comment:
Is this for future use?

Reply:
Yea, every stream has an id, this value is not used in the stream itself, but I like to have it there to notice that there is a stream-id bound to it.